From 639a22f62d47ae87844f75e8e77f6e4076a305b5 Mon Sep 17 00:00:00 2001
From: Xiaolei Wang <xiaolei.wang@windriver.com>
Date: Sun, 3 May 2020 16:54:44 +0800
Subject: [PATCH] ASoC: ti: set davinci_mcasp config for omap2

am3358 evmssk need davinci_mcasp load tlv320aic3x for audio

Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 sound/soc/ti/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/ti/Kconfig b/sound/soc/ti/Kconfig
index 2197f3e1eaed..340614795c4e 100644
--- a/sound/soc/ti/Kconfig
+++ b/sound/soc/ti/Kconfig
@@ -168,7 +168,7 @@ config SND_SOC_OMAP_OSK5912
 
 config SND_SOC_DAVINCI_EVM
 	tristate "SoC Audio support for DaVinci EVMs"
-	depends on ARCH_DAVINCI && I2C
+	depends on (ARCH_DAVINCI || ARCH_OMAP2PLUS) && I2C
 	select SND_SOC_DAVINCI_ASP if MACH_DAVINCI_DM355_EVM
 	select SND_SOC_DAVINCI_ASP if SND_SOC_DM365_AIC3X_CODEC
 	select SND_SOC_DAVINCI_VCIF if SND_SOC_DM365_VOICE_CODEC
-- 
2.17.1

